src/basicOp.js

```diff
@@ -24,7 +24,22 @@
  * @return {number}
  */
 function basicOp(operation, value1, value2) {
-  // write code here
+  switch (operation) {
+    case '+' :
```

Comment:
пробел перед двоеточием не нужен
```suggestion
    case '+':
```